cpp/include/cudf_test/timestamp_utilities.cuh

```diff
@@ -55,7 +55,6 @@ inline cudf::test::fixed_width_column_wrapper<T, int64_t> generate_timestamps(in
   auto lhs = start.time_since_epoch().count();
   auto rhs = stop.time_since_epoch().count();
 
-  // When C++17, auto [min, max] = std::minmax(lhs, rhs)
   auto min   = std::min(lhs, rhs);
   auto max   = std::max(lhs, rhs);
```

Comment:
`When C++17, auto [min, max] = std::minmax(lhs, rhs)` => so why not `auto [min, max] = std::minmax(lhs, rhs)` now?

Reply:
You get: `error: structured binding cannot be captured` which from googling seems to be a compiler bug?

Reply:
Oh no...